src/core/device_model.jl

```diff
@@ -16,21 +16,33 @@ function _check_device_formulation(
 end
 
 """
+    DeviceModel(
+        ::Type{D},
+        ::Type{B},
+        feedforwards::Vector{<:AbstractAffectFeedforward}
+        use_slacks::Bool,
+        duals::Vector{DataType},
+        services::Vector{ServiceModel}
+        attributes::Dict{String, Any}
+    )
+
 Establishes the model for a particular device specified by type. Uses the keyword argument
 feedforward to enable passing values between operation model at simulation time
 
 # Arguments
 
--`::Type{D}`: Power System Device Type
--`::Type{B}`: Abstract Device Formulation
-
-# Accepted Key Words
-
-  - `feedforward::Array{<:AbstractAffectFeedforward}` : use to pass parameters between models
+  - `::Type{D} where D<:PSY.Device`: Power System Device Type
+  - `::Type{B} where B<:AbstractDeviceFormulation`: Abstract Device Formulation
+  - `feedforward::Array{<:AbstractAffectFeedforward} = Vector{AbstractAffectFeedforward}()` : use to pass parameters between models
+  - `use_slacks::Bool = false` : Add slacks to the device model
```

Comment:
Mention, the slack implementation is model dependent. Not all models have slack implementations